### ISSUE TITLE
training loss

### ISSUE THREAD

**Issue by luoyuchenmlcv** [Jun-CEN/Open_world_3D_semantic_segmentation#10]
Hi, thanks for presenting your amazing work!

In original code base of Cylinder3D, the training loss is lovasz loss + weighted CE loss.

In your code base, I see only training naive and training MC dropout align with this training loss, but training with RCF and RCF+synthesis both do not use  lovasz loss, in validation process lovasz loss are still used.

Could you explain why is this case?

**Comment by Jun-CEN** [comment]
We only use classical CE loss and find the performance is similar with additional lovasz loss, so we do not use lovasz loss in our method.

**Comment by luoyuchenmlcv** [comment]
Thanks for that explanation!
